chunks/chunks.go

```diff
@@ -299,33 +301,30 @@ func (w *Writer) WriteChunks(chks ...Meta) error {
 		}
 	}
 
-	var (
-		b   = [binary.MaxVarintLen32]byte{}
-		seq = uint64(w.seq()) << 32
-	)
+	var seq = uint64(w.seq()) << 32
 	for i := range chks {
 		chk := &chks[i]
 
 		chk.Ref = seq | uint64(w.n)
 
-		n := binary.PutUvarint(b[:], uint64(len(chk.Chunk.Bytes())))
+		n := binary.PutUvarint(w.buf[:], uint64(len(chk.Chunk.Bytes())))
```

Comment:
Curious what the difference b/w passing `w.buf` and `w.buf[:]` is...

Reply:
`w.buf` is an array, but `PutUvarint` expects a slice. `w.buf[:]` would convert an array into a slice.

Reply:
Nice!